src/main/java/edu/metrostate/ics499/team2/services/ElementService.java

```diff
@@ -37,5 +37,155 @@ public ElementService(ElementRepository elmRepo) {
+		// period 6
+		elmRepo.save(new Element("Casium", 			"Cs", 	"Metal", 		55, 132.9, 0, 0));
+		elmRepo.save(new Element("Barium", 			"Ba", 	"Metal", 		56, 137.3, 0, 0));
+		elmRepo.save(new Element("Lanthanum",		"La", 	"Metal", 		57, 138.9, 0, 0));
+
+		elmRepo.save(new Element("Hafnium", 		"Hf", 	"Metal", 		72, 178.5, 0, 0));
+		elmRepo.save(new Element("Tantalum",		"Ta", 	"Metal", 		73, 180.9, 0, 0));
+		elmRepo.save(new Element("Tungsten",		"W",	"Metal", 		74, 183.8, 0, 0));
+		elmRepo.save(new Element("Rhenium",			"Re", 	"Metal", 		75, 186.2, 0, 0));
+		elmRepo.save(new Element("Osmium", 			"Os", 	"Metal", 		76, 190.2, 0, 0));
+		elmRepo.save(new Element("Iridium", 		"Ir", 	"Metal", 		77, 192.2, 0, 0));
+		elmRepo.save(new Element("Platinum",		"Pt", 	"Metal", 		78, 195.1, 0, 0));
+		elmRepo.save(new Element("Gold", 			"Au", 	"Metal", 		79, 197.0, 0, 0));
+		elmRepo.save(new Element("Mercury",			"Hg", 	"Metal", 		80, 200.6, 0, 0));
+		elmRepo.save(new Element("Thallium",		"Tl", 	"Metal", 		81, 204.4, 0, 0));
+		elmRepo.save(new Element("Lead", 			"Pb", 	"Metal", 		82, 207.2, 0, 0));
+		elmRepo.save(new Element("Bismuth",			"Bi", 	"Metal", 		83, 209.0, 0, 0));
+		elmRepo.save(new Element("Polonium", 		"Po", 	"Metalloid", 	84, (209), 0, 0));
+		elmRepo.save(new Element("Astatine", 		"At", 	"Metalloid", 	85, (210), 0, 0));
+		elmRepo.save(new Element("Radon", 			"Rn", 	"Nonmetal", 	86, (222), 0, 0));
+		// period 7
+		elmRepo.save(new Element("Francium",		"Fr", 	"Metal", 		87, (223), 0, 0));
+		elmRepo.save(new Element("Radium", 			"Ra", 	"Metal", 		88, (226), 0, 0));
+		elmRepo.save(new Element("Actinium",		"Ac", 	"Metal", 		89, (227), 0, 0));
+
+		elmRepo.save(new Element("Rutherfordium",	"Rf", 	"Metal", 		104, (267), 0, 0));
+		elmRepo.save(new Element("Dubnium", 		"Db", 	"Metal", 		105, (268), 0, 0));
+		elmRepo.save(new Element("Seaborgium", 		"Sg", 	"Metal", 		106, (271), 0, 0));
+		elmRepo.save(new Element("Bohrium", 		"Bh", 	"Metal", 		107, (272), 0, 0));
+		elmRepo.save(new Element("Hassium", 		"Hs", 	"Metal", 		108, (270), 0, 0));
+		elmRepo.save(new Element("Meitnerium", 		"Mt", 	"Metal", 		109, (276), 0, 0));
+		elmRepo.save(new Element("Darmstadtium", 	"Ds",	"Metal", 		110, (281), 0, 0));
+		elmRepo.save(new Element("Roentgenium",		"Rg", 	"Metal", 		111, (280), 0, 0));
+		elmRepo.save(new Element("Copernicium",		"Cn", 	"Metal", 		112, (285), 0, 0));
+		elmRepo.save(new Element("unknown", 		"Nh",  	"Metal", 		113, (284), 0, 0));
+		elmRepo.save(new Element("unknown", 		"Fl", 	"Metal", 		114, (289), 0, 0));
+		elmRepo.save(new Element("unknown", 		"Mc",  	"Metal", 		115, (289), 0, 0));
+		elmRepo.save(new Element("unknwon", 		"Lv", 	"Metal", 		116, (293), 0, 0));
+		elmRepo.save(new Element("unknown", 		"Ts", 	"Metal", 		117, (294), 0, 0));
+		elmRepo.save(new Element("unknown", 		"Og", 	"Nonmetal", 	118, (294), 0, 0));
+        System.out.println("Data creation complete...");
+    }
```

Comment:
Good call moving this out to a service. We want to keep the Main method as clean as possible.